entity-framework/core/what-is-new/ef-core-7.0/breaking-changes.md

```diff
@@ -131,11 +131,21 @@ The performance improvements linked to the new method are significant enough tha
 
 #### Mitigations
 
-You can let EF Core know that the target table has a trigger; doing so will revert to the previous, less efficient technique. This can be done by configuring the corresponding entity type as follows:
+Starting with EF Core 8.0, the use or not of the "OUTPUT" clause can be configured explicitly. For example:
+
+```csharp
+protected override void OnModelCreating(ModelBuilder modelBuilder)
+{
+    modelBuilder.Entity<Blog>()
+        .ToTable(tb => tb.UseSqlOutputClause(false));
```

Comment:
@ajcvickers we still only show the convention below for adding triggers to all tables (rather than using the new UseSqlOutputClause() mechanism) - I'm wondering whether we're missing functionality or something for doing UseSqlOutputClause() from a convention (or should we just doc it?)